WordPress/Jetpack/WidgetConfiguration.swift

```diff
@@ -0,0 +1,39 @@
+// Jetpack Widget configuration
+
+import Foundation
+
+@objc extension AppConfiguration {
+
+    @objc(AppConfigurationWidget)
+    class Widget: NSObject {
+        @objc(AppConfigurationWidgetStats)
+        class Stats: NSObject {
+            @objc static let keychainTokenKey = "OAuth2Token"
```

Comment:
@staskus I'm working on doing something similar to the constants used by Extensions. Is there a reason for opting to use the same token key for both apps?

Reply:
I didn't find it necessary, since it was used in tandem with `keychainServiceName` which was different for both apps. Did you encounter any problems with that?